README.md

```diff
@@ -4,81 +4,74 @@
   ![Master branch unit testing result](https://travis-ci.org/hackcambridge/hack-cambridge-website.svg?branch=master)
 ](https://travis-ci.org/hackcambridge/hack-cambridge-website)
 
-## Getting Started
+## Getting started
 
-To run the website on your machine, make sure you have [Node.js](https://nodejs.org) and [Yarn](https://yarnpkg.com/) installed. Then inside this folder, run
+To run the website on your machine, first make sure you have [Node.js](https://nodejs.org), [Yarn](https://yarnpkg.com/), and [Docker](https://www.docker.com) installed.  Then [clone this repository from GitHub](https://help.github.com/articles/cloning-a-repository/):
 
-```bash
-yarn global add gulp
-yarn install
-gulp serve
+```
+git clone https://github.com/hackcambridge/hack-cambridge-website.git
 ```
 
-And your server will be ready to go.
+### Environment variables
 
-## Environment Variables
+Certain environment variables need to be available for features to work. These variables are stored in the `.env` file at the root of the repository.
 
-Certain environment variables need to be available for features to work. For convenience
-you can do this in the `.env` file at the root of your project.
+To get started, create a `.env` file at the root of your project with the following contents:
 
 ```
-MAILCHIMP_API_KEY=
-MAILCHIMP_INTERESTED_LIST_ID=
+AUTH_SESSION_SECRET=auth_session_secret_placeholder
+PGDATABASE=postgres
+PGHOST=127.0.0.1
+PGPASSWORD=
+PGPORT=5432
+PGUSER=postgres
+S3_BUCKET=s3_bucket_placeholder
+APPLICATIONS_OPEN_STATUS=
 APPLICATION_URL=
-TEAM_APPLICATION_URL=
-STRIPE_PUBLISH_KEY=
-STRIPE_PRIVATE_KEY=
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
 GOOGLE_SHEETS_AUTH_EMAIL=
 GOOGLE_SHEETS_AUTH_KEY=
 GOOGLE_SHEETS_WIFI_SHEET_ID=
+MAILCHIMP_API_KEY=
+MAILCHIMP_INTERESTED_LIST_ID=
+MAILGUN_API_KEY=
 MYMLH_CLIENT_ID=
 MYMLH_CLIENT_SECRET=
-AUTH_SESSION_SECRET="auth_session_secret_placeholder"
-MAILGUN_API_KEY=
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-S3_BUCKET="s3_bucket_placeholder"
-PGHOST=
-PGUSER=
-PGDATABASE=
-PGPASSWORD=
-PGPORT=
-SLACK_API_TOKEN=
-APPLICATIONS_OPEN_STATUS=
 PUSHER_KEY=
+SLACK_API_TOKEN=
+STRIPE_PRIVATE_KEY=
+STRIPE_PUBLISH_KEY=
+TEAM_APPLICATION_URL=
 ```
 
-In order to get the basic website up-and-running none of these variables need to
-be set, but the .env file does need to be present. Not all features will work
-without valid tokens for all the environment variables.
+Leaving most of these variables undefined is sufficient to get the basic website up-and-running, but not all features will work without valid tokens for all the environment variables.
```

Comment:
Though this is a public repo, it might be good to explain how to get the variables

Reply:
We don't yet have a good process for this, for now it's just asking on Slack.

Reply:
Can we put that in?